src/DataCallout/index.tsx

```diff
@@ -0,0 +1,121 @@
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+/** @jsxImportSource @emotion/react */
+
+import { ComponentType } from 'react';
+import { Button } from '../Button';
+import { Icon } from '../Icon';
+import { UikitIconNames } from '../Icon/icons';
+import { Link } from '../Link';
+import { Typography } from '../Typography';
+import { ThemeColorNames } from '../theme/types';
+import { css, useTheme } from '..';
+
+export const CircleContainer: ComponentType<{
+  fill: keyof ThemeColorNames;
+  children: React.ReactNode;
+}> = ({ fill, children }) => {
+  const theme = useTheme();
+  return (
+    <div
+      css={css`
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 44px;
+        width: 44px;
+        background-color: ${theme.colors[fill]};
+        border-radius: 100%;
+      `}
+    >
+      {children}
+    </div>
+  );
+};
+
+export const DataCallout: ComponentType<{
+  iconName: UikitIconNames;
+  iconFill: keyof ThemeColorNames;
+  circleFill: keyof ThemeColorNames;
+  title: string;
+  urlData: { text: string; href: string };
+  children: React.ReactNode;
+  className?: string;
+}> = ({ iconName, iconFill, circleFill, title, urlData, children, className }) => {
+  return (
+    <div
+      css={css`
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: start;
+        text-align: center;
+        height: 100%;
+        padding: 0 5%;
+      `}
+      className={className}
+    >
+      <CircleContainer fill={circleFill}>
+        <Icon name={iconName} fill={iconFill} />
+      </CircleContainer>
+      <Typography
+        color="primary"
+        variant="subtitle"
+        css={css`
+          margin-top: 16px;
+          margin-bottom: 5px;
+        `}
+        as="h2"
+      >
+        {title}
+      </Typography>
+      <Typography
+        as="p"
+        css={css`
+          margin: 14px 0;
+        `}
+      >
+        {children}
+      </Typography>
+
+      <Link
+        href={urlData.href}
+        underline={false}
+        css={css`
+          margin: 0 15px;
+          margin-top: auto;
+        `}
+        target="_blank"
+      >
+        <Button css={css({ padding: '6px 12px', marginBottom: '20px' })}>
```

Comment:
is there a reason to use different css syntax in Link versus Button? if not we should try to be consistent in case there's API changes in emotion later.

Reply:
Most weird looking things in this PR are just low hanging fruit was fixed as I copy/pasted this code from platform-ui with some tweaks, over dumping it directly into a new project. So in short no reason, the `css` object syntax works best for basic properties because you get typechecking (woo). 

Will address the `css` function calls for consistency - nice catch!

Reply:
good to know about typechecking. this is ringing a bell for me from a recent conversation about CSS syntax actually...